modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/utils/getFields.ts

```diff
@@ -51,14 +52,18 @@ const validSchemaPropertyFilter = (propertyKey: string) => {
 
 function getValidFields({
 	contextPath,
+	parentPath,
 	schemaName,
 	schemas,
 	visitedFields,
+	xFilterable = [],
```

Comment:
To clarify code a bit, can we call this `filterablePaths`? x is just a bit cryptic.